.env

```diff
@@ -1,12 +1,13 @@
 VUE_APP_WALLET_URL=https://wallet.superhero.com
 VUE_APP_JITSI_HOST=meet.aepps.com
-VUE_APP_BACKEND_URL=https://raendom-backend.z52da5wt.xyz
-VUE_APP_NODE_URL=https://mainnet.aeternity.io
-VUE_APP_EXPLORER_URL=https://mainnet.aeternal.io
+VUE_APP_BACKEND_URL=https://test-tipping.aeternity.art
+VUE_APP_NODE_URL=https://testnet.aeternity.io
+VUE_APP_MIDDLEWARE_URL=https://testnet.aeternity.io/mdw
+VUE_APP_EXPLORER_URL=https://testnet.aeternal.io
 VUE_APP_COMPILER_URL=https://latest.compiler.aepps.com
-VUE_APP_CONTRACT_V1_ADDRESS=ct_2AfnEfCSZCTEkxL5Yoi4Yfq6fF7YapHRaFKDJK3THMXMBspp5z
-VUE_APP_CONTRACT_V2_ADDRESS=
-VUE_APP_CONTRACT_V3_ADDRESS=ct_2Hyt9ZxzXra5NAzhePkRsDPDWppoatVD7CtHnUoHVbuehwR8Nb
-VUE_APP_WORD_REGISTRY_ADDRESS=ct_2cYtnXKZE5BhuRMstJBEzvfmdZkrE5LooomcZ6XXzjfyEZPXCq
+VUE_APP_CONTRACT_V1_ADDRESS=ct_2Cvbf3NYZ5DLoaNYAU71t67DdXLHeSXhodkSNifhgd7Xsw28Xd
+VUE_APP_CONTRACT_V2_ADDRESS=ct_2ZEoCKcqXkbz2uahRrsWeaPooZs9SdCv6pmC4kc55rD4MhqYSu
+VUE_APP_CONTRACT_V3_ADDRESS=ct_WscpdLQf6ZZxoVqrsEwUwmuAEdzEkJii5W5TzG84rVgHeK6BW
+VUE_APP_WORD_REGISTRY_ADDRESS=ct_2sHzycCWkiyfZkCMoprn3Q7U8SQh3FYXp6iB1RAFVJy1pNWZAY
```

Comment:
This is going to replace the `mainnet` config, can we think of another way to switch between testnet and mainnet setup.

Reply:
yes needs to be replaced before merge, but otherwise not testable currently

Reply:
9257d068 In local development we can easily switch between different .env files. The drawback is I don't know how they can be switched in the preview of the branch deployed by gh action bot.

Reply:
Actually, wouldn't be better .env to be with the Testnet data and to introduce .env.prod for Mainnet so by default we'll use in local mode and the branch previews Testnet and won't be messing with the Mainnet data.

Reply:
good solution